LoM-Install/arista/do-install.py

```diff
@@ -0,0 +1,768 @@
+        else:
+            logger.log("Error: Unix eAPI protocol is not running.")
+            return False, None
+
+        # Create an instance of AristaSwitchEAPIHelper to execute commands via eAPI
+        g_switch_eapi_handler = eapi_helper.AristaSwitchEAPIHelper()
+        g_switch_eapi_handler.connect()
+        logger.log("Created an instance of AristaSwitchEAPIHelper")
+
+    except Exception as e:
+        logger.log("Error in setup: {}".format(e))
+        raise e
+    
+    return True, existing_install_config
+
+def start_installation(installation_path):
+    """
+    This function is responsible for starting the installation process.
+
+    It performs the following steps:
+    1. Setup the installation
+    2. Perform pre-checks
+    3. Extract and log daemon information
+    4. Cleanup any existing daemons
+    5. Perform common checks before installation
+    6. Start the daemons in the management namespace
+    7. Validate if all daemons are running
+    8. Perform common checks after installation
+    9. Validate the common checks
+    10. Perform post-checks
+
+    Parameters:
+    installation_path (str): The path where the installation should be set up.
+
+    Returns:
+    bool: True if the installation was successful, False otherwise.
+    existing_config: The existing configuration if the installation was successful, None otherwise.
+    """
+    try:
+        logger.log("Starting installation at {}".format(installation_path))
+
+        # Create temporary file to indicate that the installation is being triggered by an external process (like FUSE)
+        # and not via the event handler(after reboot or after config/device repave). This is used to skip certain steps in the
+        # installation process when its triggered via the event handler.
+        with open(LOM_TEMP_INSTALLATION_FILE, 'w') as f:
+            pass
+
+        # Setup the installation
+        status, existing_config = setup(installation_path)
+        if not status:
+            logger.log("Error: Failed to setup the installation")
+            return False, None 
+        logger.log("Setup successful") 
+        
+        # Event handler support. After reboot, the event handler is triggered and it calls install_from_event_handler() function.
+        if EVENT_HANDLER_SUPPORT:            
+            # Add event Handler 
+            g_switch_cli_handler.add_boot_event_handler(handler_name=EVENT_HANDLER_NAME,
+                                                        script_path=EVENT_HANDLER_SCRIPT_PATH, 
+                                                        delay = EVENT_HANDLER_DELAY, timeout=EVENT_HANDLER_TIMEOUT,
+                                                          asynchronous=EVENT_HANDLER_ASYNCHRONOUS)
+            #verify if the event handler is added
+            event_handler_data = g_switch_cli_handler.get_event_handler_data(EVENT_HANDLER_NAME)
+            if not event_handler_data:
```

Comment:
Add To-Do comment : Add else case and add log if event handler support isnot there. 
also combine both checks.

Reply:
added